lib/helpers.ts

```diff
@@ -1,10 +1,14 @@
 import striptags from 'striptags';
+import { Lang, Post } from './types';
 
 export function paginate<T>(arr: Array<T>, whichPage: number, perPage: number): Array<T> {
   return arr.slice((whichPage - 1) * perPage, whichPage * perPage);
 }
 
-// todo: either replace all use cases with custom descriptions or make this better
+export function description(post: Post<Lang>): string {
+  return post.description || getExcerpt(post.content, 300);
```

Comment:
question: did jason finish adding custom descriptions to all his posts?  if so, i could go into cosmic and mark that field as _required_ then we could change the type to not be optional, and get rid of the fallback stuff.

Reply:
Yep he did, so maybe that would be good